notary/server/src/server.rs

```diff
@@ -228,15 +228,18 @@ async fn load_notary_signing_key(config: &NotarySigningKeyProperties) -> Result<
 }
 
 /// Read a PEM-formatted file and return its buffer reader
-pub async fn read_pem_file(file_path: &str) -> Result<BufReader<StdFile>> {
+pub async fn read_pem_file(file_path: &Option<String>) -> Result<BufReader<StdFile>> {
```

Comment:
keep this "Option handling" logic outside of these specific functions; the logic can be handled outside [here](https://github.com/tlsnotary/tlsn/blob/dev/crates/notary/server/src/server.rs#L61-L65)